test/lint/lint-python-dead-code-whitelist

```diff
@@ -0,0 +1,93 @@
+_.posix_path  # unused attribute (contrib/macdeploy/custom_dsstore.py:45)
```

Comment:
:+1: for moving these to one per line. I think you can keep the original `vulture` invocation by making this a list and then running:

`vulture --ignore-names=$(cat lint-python-dead-code-whitelist | tr '\n' ',')` or similar.

Reply:
I think it wastes less time if we just keep the autogenerated file as is